config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
+  get 'volunteers/get'
```

Comment:
You should point this route to the controller designated to handle this request.
`get 'volunteers', to: 'volunteers#get'`, it's more standard practice for routes to be written this way